src/dynarec/rv64/rv64_emitter.h

```diff
@@ -108,8 +108,10 @@ f28–31  ft8–11  FP temporaries                  Caller
 #define SPLIT20(A)  (((A)+0x800)>>12)
 #define SPLIT12(A)  ((A)&0xfff)
 
-// MOVE64x is quite complex, so use a function for this
+// MOV64x/MOV32w is quite complex, so use a function for this
 #define MOV64x(A, B)    rv64_move64(dyn, ninst, A, B)
+#define MOV32w(A, B)    rv64_move32(dyn, ninst, A, B)
```

Comment:
`rv64_move32'...)` does not handle zeroup, but `rev64_move64(...)` does, that why I didn't created `MOV32w(...)` before, there is no gain in having different macro.